src/Connection.cpp

```diff
@@ -134,23 +135,44 @@ int Connection::connectPoll(int fd, struct addrinfo* ai_ptr) {
         pollfd_t pollfds[n_fds];
         pollfds[0].fd = fd;
         pollfds[0].events = POLLOUT;
+        // clean errno
+        errno = 0;
```

Comment:
not necessary?

Reply:
@tclh123 https://github.com/douban/libmc/blob/master/include/Common.h#L61 log will use errno，if not clean here。such as EINPROGRESS EALREADY above will show in the log，it's noisy。